elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
+    """
+    Peristimulus Time Histogram (PSTH) of a list of :attr:`neo.SpikeTrain`
```

Comment:
No, I think we should stick with "psth" because it is well known. "Peristimulus" implies that there is a stimulus, and that the histogram is relative to the onset time of that stimulus. This is why I think we should use a name different from `t_start` for the stimulus onset time (to make it clear this is different from `t_start` of the spike trains), and why the argument should be required.